populate_casgenes.py

```diff
@@ -18,7 +20,6 @@
 
 '''
```

Comment:
Merge this docstring with the first one?

Reply:
Sure, no problem.